data/remnant missions.txt

```diff
@@ -1051,7 +1051,7 @@ mission "Remnant: Salvage 4"
 		payment 250000
 		set "remnant: failed salvage 4"
 		set "Remnant: Salvage 4: done"
-		dialog `You are met by the prefect who thanks who looks upset. "The ship was supposed to be disabled so its contents could be salvaged!" he signs angrily. Nonetheless, he shoves a credit chip worth <payment> into your hand. "That's half what you would have earned if the ship had been disabled." He pauses and shakes his head. "I think Tali might have some more work for you in the spaceport, but in the meantime, why don't you go find some Korath to practice disabling? It is a valuable skill among the Remnant."`
+		dialog `You are met by the prefect who looks upset. "The ship was supposed to be disabled so its contents could be salvaged!" he signs angrily. Nonetheless, he shoves a credit chip worth <payment> into your hand. "That's half what you would have earned if the ship had been disabled." He pauses and shakes his head. "I think Tali might have some more work for you in the spaceport, but in the meantime, why don't you go find some Korath to practice disabling? It is a valuable skill among the Remnant."`
```

Comment:
You also didn't change this to "a prefect" as below.

Reply:
Ah, I wasn't intending to switch all them to being "ah," but sure, I can change that. I was thinking at least sometimes the player might recognize them as being the same one.

Reply:
Well given that it's either this dialog or the one below, they should use the same language, which means either "a prefect" in both cases, or a more identifiable title aside from just "the prefect," seeing as how the player has dealt with multiple prefects at this point.

Reply:
Heck, you could actually name this prefect. Why not Prefect Chilia? I'd think that Chilia would be a military prefect given that they were the one who contacted you when you were assumed to be hostile during the first contact. Just introduce the military prefect as Chilia earlier on.

Reply:
Sure I could do that. As you said in the discord, do a PR'd change afterwards